### DISCUSSION TITLE
blog/accessibilite-profite-a-tout-le-monde/

### DISCUSSION THREAD

**Discussion by giscus** [svandegar/seraphin#10] [Announcements]
# blog/accessibilite-profite-a-tout-le-monde/

Comment rendre le web accessible à toutes et tous? Dans cet article, je partage avec vous ce que j'ai retenu de la lecture de du livre “Accessibility for everyone“, écrit par Laura Kalbag.

https://seraphin.dev/blog/accessibilite-profite-a-tout-le-monde/

**Comment by Lyokolux** [comment]
Petite coquille, dans la partie "HTML Sémantique", la balise de navigation est `<nav>` au lieu de `<navigation>`. Le Mozilla Developer Network https://developer.mozilla.org/fr/docs/Web/HTML/Element/nav décrit cela.

**Reply by svandegar** [reply]
Oh oui, en effet, c'est une erreur de ma part. Je vais la corriger. Merci!
